components/pageWrapper.js

```diff
@@ -1,6 +1,7 @@
 import React from 'react'
 import Nav from '../components/nav'
 import Head from '../components/head'
+import ScrollToTop from './scrolltotop'
```

Comment:
looks like the deploy is failing because it can't find scrollToTop

should this be import ScrollToTop from '../components/ScrollToTop.js' ?

Reply:
@joelwass nope. It's import ScrollToTop from './scrolltotop'.

Reply:
You can run `next build` locally to see if it builds, but remotely it's failing due to that Import I believe